src/editor.rs

```diff
@@ -910,8 +918,8 @@ mod tests {
         }
         editor.delete_char();
         assert_eq!(editor.rows[0].chars, "Hello".as_bytes());
-        editor.move_cursor(&AKey::Left);
-        editor.move_cursor(&AKey::Left);
+        editor.move_cursor(&AKey::Left, false);
+        editor.move_cursor(&AKey::Left, false);
         editor.delete_char();
         assert_eq!(editor.rows[0].chars, "Helo".as_bytes());
     }
```

Comment:
Could you add a test for the new behavior?

Reply:
Sure! I added it.